.babelrc

```diff
@@ -1,5 +1,5 @@
 {
     "presets": [
-        "es2015-loose"
+        "es2015"
```

Comment:
What's wrong with loose? Changing this makes all scripts larger by requiring unused to code in transpiled class constructors.